Source/AppAuth/iOS/OIDAuthState+IOS.h

```diff
@@ -50,6 +50,31 @@ NS_ASSUME_NONNULL_BEGIN
                      presentingViewController:(UIViewController *)presentingViewController
                                      callback:(OIDAuthStateAuthorizationCallback)callback;
 
+/*! @brief Convenience method to create a @c OIDAuthState by presenting an authorization request
+        (optionally using an emphemeral browser session that shares no cookies or data with the
+        normal browser session) and performing the authorization code exchange in the case of code
+        flow requests. For the hybrid flow, the caller should validate the id_token and c_hash, then
+        perform the token request (@c OIDAuthorizationService.performTokenRequest:callback:)
+        and update the OIDAuthState with the results (@c
+        OIDAuthState.updateWithTokenResponse:error:).
+    @param authorizationRequest The authorization request to present.
+    @param presentingViewController The view controller from which to present the
+        @c SFSafariViewController. On iOS 13, the window of this UIViewController
```

Comment:
I think it's reasonable to omit the specifics from this method's comment since the actual implementation of all of that logic is limited to the OIDExternalUserAgent* class which these invoke.